### PR TITLE
Extend AtomsBaseTesting and AtomsBase test routines

### PR DESCRIPTION
**Pull request by mfherbst**
- Reintroduce some previously disabled test
- Various smaller fixes to AtomsBase
- Fixes to AtomsBaseTesting

@cortner @rkurchin Could some of you take a quick look ?

### CODE REVIEW

File: test/flexible_system.jl [hunk 1]
```diff
@@ -0,0 +1,57 @@
+using AtomsBase
+using Unitful
+using UnitfulAtomic
+using Test
+
+@testset "Fast system" begin
```
Comment (cortner) [inline test/flexible_system.jl:6]: Should this be "Flexible System" ?

File: src/utils/show.jl [hunk 2]
```diff
@@ -18,7 +18,9 @@ function show_system(io::IO, system::AbstractSystem{D}) where {D}
     print(io, ")")
 end
 
-function show_system(io::IO, ::MIME"text/plain", system::AbstractSystem{D}) where {D}
+# TODO Put the show_ascii behind Preferences
+function show_system(io::IO, ::MIME"text/plain", system::AbstractSystem{D};
+                     show_ascii=true) where {D}
```
Comment (cortner) [inline src/utils/show.jl:23]: I vote again to make the default `false`. 87% of the time the ascii plot is not useful I think.

**Review by cortner — APPROVED** [review]

File: src/utils/show.jl (hunk 2 above)
Comment (mfherbst) [reply src/utils/show.jl:23]: Happy to do that. If it's a preference I can activate it for myself. Other thoughts ?

We could also have a "denseness limit" e.g. only show if less than 10 atoms or so or if less than 10% of the image will be covered by strings ... but again, should be made configurable.

Comment (mfherbst) [reply src/utils/show.jl:23]: @rkurchin @jgreener64

Comment (mfherbst) [reply src/utils/show.jl]: I reverted that change here. Will open follow-up PR to start discussion in #122.

**Pull request merged**